src/sentry/conf/server.py

```diff
@@ -2923,3 +2923,7 @@ def build_cdc_postgres_init_db_volume(settings):
 #   },
 # }
 SLICED_KAFKA_TOPICS: Mapping[Tuple[str, int], Mapping[str, Any]] = {}
+
+# Used by silo tests -- when requests pass through decorated endpoints, switch the server silo mode to match that
+# decorator.
+SINGLE_SERVER_SILO_MODE = False
```

Comment:
Acceptance tests act by rendering a full react view and allowing that react view to query many region and control endpoints for data to render before a screenshot.

Unfortunately, our current system assumes every process has exactly one silo mode at any given time, including tests.  This new option allows acceptance tests, specifically, to "swap out" the current silo mode in a thread safe manner.  A future system may involve acceptance tests running against two separate processes to remove the strict need for this logic and to better fully simulate and test the RPC experience.